### PR TITLE
[19.03 backport] docs: add redirect for old location of daemon reference

### PR DESCRIPTION
**Pull request by thaJeztah**
backport of https://github.com/docker/cli/pull/2351

### CODE REVIEW

**Comment by thaJeztah** [comment]
@silvin-lubecki this is ready for review now

File: docs/reference/commandline/dockerd.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: "dockerd"
 aliases: ["/engine/reference/commandline/daemon/"]
```
Comment (silvin-lubecki) [inline docs/reference/commandline/dockerd.md]: I think this one has been removed on the master branch 😅

Comment (thaJeztah) [reply docs/reference/commandline/dockerd.md]: Good catch; guess I forgot to update it after I did on master. Updating now

**Comment by thaJeztah** [comment]
Updated 👍

**Pull request merged**